.circleci/config.yml

```diff
@@ -22,6 +22,9 @@ workflows:
       - test-lint:
           requires:
             - prep-deps
+      - test-lint-shellcheck:
+          requires:
+            - prep-deps
```

Comment:
Could this be run before dependencies are installed? I think that would work fine.